lib/adminer.php

```diff
@@ -21,4 +21,51 @@ function databases($flush = true)
     function databasesPrint($missing)
     {
     }
+
+    function tableStructurePrint($fields)
+    {
+        if (class_exists(rex_sql_schema_dumper::class)) {
+            $table = rex_sql_table::get($_GET['table']);
```

Comment:
brauchts auch hier noch nen `class_exists` check für `rex_sql_table`?

Reply:
die klasse gibts seit rex5.0... in der package.yml diese addons fehlt noch die min-rex version.

der check ist also nicht nötig.